scripts/sync-csi-proxy.ps1

```diff
@@ -13,7 +13,7 @@ Copy-Item -Path "C:\Users\$env:UserName\csi-proxy.exe" -Destination "C:\etc\kube
 
 # restart the csiproxy service
 $flags = "-v=5 -windows-service -log_file=C:\etc\kubernetes\logs\csi-proxy.log -logtostderr=false"
-sc.exe create csiproxy binPath= "C:\etc\kubernetes\node\bin\csi-proxy.exe $flags"
+sc.exe create csiproxy start= "auto" binPath= "C:\etc\kubernetes\node\bin\csi-proxy.exe $flags"
```

Comment:
I've recently removed this file in https://github.com/kubernetes-csi/csi-proxy/pull/190, I'll keep an eye on this PR if it gets merged before that one to use this flag in the new file too